docs/UserGuide.md

```diff
@@ -7,152 +7,255 @@
+**Format:**
+```
+exit
+```
+* Closes the application
+* All data is automatically saved
+> ⚠️ **Warning:** No confirmation message is displayed. 
 
-### Viewing help : `help`
+### Editing The Data File
 
-Shows a message explaining how to access the help page.
+LegacyLink data are saved automatically as a JSON file `[JAR file location]/data/addressbook.json and [JAR file location]/data/eventbook.json`. Advanced users are welcome to update data directly by editing that data file.
 
-Format: `help`
+<box type="warning" seamless>
 
-![help message](images/helpMessage.png)
+**Caution:**
+If your changes to the data file makes its format invalid, LegacyLink will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
+Furthermore, certain edits can cause the LegacyLink to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
+
+</box>
 
-### Listing all persons : `list -p`
 
-Shows a list of all persons in the address book.
 
-Format: `list -p`
+## Managing Contacts
+
+### View Your Contact List 👥
+```
+list -p
+```
+* Shows all your family contacts in one view
+* Use this to check contact details or before using other commands
+
+![ListPersonCommand.png](images/ListPersonCommand.png)
 
 <box type="tip" seamless>
 
 **Tip:** Preceding, trailing and intermediate whitespaces will still result in a successful command.
 </box>
 
-![ListPersonCommand.png](images/ListPersonCommand.png)
+### Add a New Contact ➕
+**Format:**
+```
+add -n NAME -p PHONE_NUMBER -e EMAIL -rs RELATIONSHIP
+```
+* Adds a person to the address book.
+* All parameters `NAME`, `PHONE_NUMBER`, `EMAIL`, `RELATIONSHIP` must be present.
+* If you want to use special characters such as `d/o` or `s/o` in `name`, please omit the `/` and enter `so` and `do` instead.
 
-### Adding a person: `add`
+<box type="tip" seamless>
 
-Adds a person to the address book.
+**Rules to Remember:**
 
-Format: `add -n NAME -p PHONE_NUMBER -e EMAIL -rs RELATIONSHIP`
+| Field       | Requirements                                                                                                    |
+|-------------|-----------------------------------------------------------------------------------------------------------------|
+| Name        | • Letters, numbers, and spaces only<br>• Cannot be blank                                                        |
+| Phone       | • 3-15 digits<br>• Alphabetical characters (A-Z, a-z) and special characters (/ - _ ( )) are permitted. <br> • At least one digit is required in the number |
+| Email       | • Must be valid email format (example@domain.com)                                                               |
+| Relationship| • Letters only<br>• Cannot be blank                                                                             |
```

Comment:
referring to issue #226, maybe we can say that the onus is on you to add meaningful relationship, since relationships can be any string of letters

Reply:
Makes sense, edited.